locales/help_enu/gkhCAIndex.html

```diff
@@ -4,22 +4,22 @@
 <meta charset="utf-8" />
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <link rel="stylesheet" href="styles.css" type="text/css" />
-<title>Authenticity index</title>
+<title>Authenticity Index</title>
 </head>
 <body>
-<h1>Authenticity index</h1>
-<p>Authenticity index of information about person is a special indicator that can be evaluated for person entity only.</p>
-<p>Each reference to a source in a person entity has specific authenticity level of mention that man in the source. Authenticity level can have the following values:</p>
+<h1>Authenticity Index</h1>
+<p>The Authenticity Index is information about a person as a special indicator that can be evaluated for a person entity only.</p>
```

Comment:
\`\`Authenticity index'' isn't information about person, I believe.  It's an attribute of data about that person.

Reply:
Generally yes, it is. And I did not catch this accent in edits. Need to reformulate. Respect!